### ISSUE TITLE
New release of vcf2cytosure

### ISSUE THREAD

**Issue by dnil** [Clinical-Genomics/MIP#649]
0.4.1 available! Now expects sex as a command line option, and produces cleaner output files that will be more manageable on the clinic side.

https://github.com/NBISweden/vcf2cytosure/releases/tag/0.4.1

**Comment by henrikstranneheim** [comment]
@dnil and @J35P312 I have tried to install the new version 0.4.1 from: https://github.com/NBISweden/vcf2cytosure/archive/0.4.1.zip. However, when I run the pip:
```
Collecting lxml (from vcf2cytosure==0.4.0)
  Using cached https://files.pythonhosted.org/packages/e5/14/f4343239f955442da9da1919a99f7311bc5627522741bada61b2349c8def/lxml-4.2.5-cp27-cp27mu-manylinux1_x86_64.whl
Collecting cyvcf2 (from vcf2cytosure==0.4.0)
Requirement already satisfied: numpy in /mnt/hds/proj/cust003/develop/modules/miniconda/envs/mip_tiddit/lib/python2.7/site-packages (from cyvcf2->vcf2cytosure==0.4.0) (1.15.4)
Installing collected packages: lxml, cyvcf2, vcf2cytosure
  Running setup.py develop for vcf2cytosure
Successfully installed cyvcf2-0.10.1 lxml-4.2.5 vcf2cytosure
```
It says version 0.4.0, which is also what is present in: https://github.com/NBISweden/vcf2cytosure/blob/master/setup.py.

When I run:
```
$vcf2cytosure --version
VCF2cytosure - convert SV vcf files to cytosure 0.3.0
```
Which is also the version that is in: https://github.com/NBISweden/vcf2cytosure/blob/master/vcf2cytosure.py.

I am utterly confused to what I have just installed?

I also noticed that it is using python 3 now. Is is backwards compatible with 2,7? We have been running it with 2.7?

**Comment by J35P312** [comment]
I have run it using 2.7 as well on my laptop!

**Comment by dnil** [comment]
Right; 0.4.2 should have both the correct version and args.sex instead.

Scary thing, putting many python programs in the same conda environment. Makes me long for statically linked binaries. ;-)

We can take a look at TIDDIT and svdb. I’m not aware of any showstopper reason not to move them to py3 as well other than some cosmetics, but it is likely not going to happen overnight. For now please go with all in pytho=2.7. Thanks!

// Daniel

**Comment by henrikstranneheim** [comment]
Yeah, those have a certain charm😁. Will leave it as is in 2.7 then. I will test 0.4.2 tomorrow.

**Comment by J35P312** [comment]
Do so, 0.4.2 will work!

I believe that in the future, we will have tools that package our scripts in "machine-code"; this code will include all dependencies and we will probably be  able to move it between computers having similar architecture!
